src/db/tables/subscribers.js

```diff
@@ -231,11 +231,12 @@ async function deleteResolutionsWithEmail (id, email) {
   })
   const { breach_resolution: breachResolution } = subscriber
   // if email exists in breach resolution, remove it
-  if (breachResolution[email]) {
+  if (breachResolution && breachResolution[email]) {
     delete breachResolution[email]
+    console.info(`Deleting resolution with email: ${email}`)
+    await setBreachResolution(subscriber, breachResolution)
   }
-
-  return await setBreachResolution(subscriber, breachResolution)
+  console.info(`No resolution with ${email} found, skip`)
```

Comment:
Wouldn't this call the "No resolution found" log for all cases?  Did you intend to add an `else`?

Reply:
Thanks! Intended to copy the `return` as well